cluster.tf

```diff
@@ -28,6 +28,12 @@ resource "google_container_cluster" "primary" {
     channel = var.release_channel
   }
 
+  enable_shielded_nodes = true
+  workload_identity_config {
+    # Currently, the only supported identity namespace is the project's default.
+    identity_namespace = "${var.project_id}.svc.id.goog"
+  }
```

Comment:
We shall update the Readme with all the new features added (shielded nodes, update pool variant...).
Also we can take advantage of this PR and upgrade all the versions (google providers, terraform 0.14.7, random provider).